frontend/src/components/ViewActivities/editActivity.js

```diff
@@ -132,41 +134,49 @@ class EditActivity extends React.Component {
+        {formElements.locationElementFormGroup( // END LOCATION
+          'formActivityEndLocation',                 // controlId
+          'End Location:',                           // formLabel
+          this.countriesDropdown(this.editEndLocRef, // defaultValue ref
+            this.editEndTzRef, // countriesDropdown tzref
+            getField(activity, DB.ACTIVITIES_END_COUNTRY)) // countriesDropdown defaultCountry
           )}
-        {formElements.dateTimeTzFormGroup(
-          'formActivityStartTime',       // controlId
-          'From:',                       // formLabel
-          this.editStartDateRef,         // dateRef
-          null,                          // dateDefault 
-          this.editStartTimeRef,         // timeRef, 
-          null,                          // timeDefault, 
-          this.timezoneDropdown('start') // tzpicker 
+        {formElements.dateTimeTzFormGroup( // START TIME
+          'formActivityStartTime',                         // controlId
+          'From:',                                         // formLabel
+          this.editStartDateRef,                           // dateRef
+          time.getISODate(getField(activity, DB.ACTIVITIES_START_TIME), 
+              getField(activity, DB.ACTIVITIES_START_TZ)), // dateDefault 
+          this.editStartTimeRef,                           // timeRef, 
+          time.get24hTime(getField(activity, DB.ACTIVITIES_START_TIME), 
+              getField(activity, DB.ACTIVITIES_START_TZ)), //timeDefault, 
+          this.timezoneDropdown('start')                   // tzpicker 
           )}
-        {formElements.dateTimeTzFormGroup(
-          'formActivityEndTime',       // controlId
-          'To:',                       // formLabel
-          this.editEndDateRef,         // dateRef
-          null,                        // dateDefault 
-          this.editEndTimeRef,         // timeRef, 
-          null,                        //timeDefault, 
-          this.timezoneDropdown('end') // tzpicker 
+        {formElements.dateTimeTzFormGroup( // END TIME
+          'formActivityEndTime',                         // controlId
+          'To:',                                         // formLabel
+          this.editEndDateRef,                           // dateRef
+          time.getISODate(getField(activity, DB.ACTIVITIES_END_TIME), 
+              getField(activity, DB.ACTIVITIES_END_TZ)), // dateDefault 
+          this.editEndTimeRef,                           // timeRef, 
+          time.get24hTime(getField(activity, DB.ACTIVITIES_END_TIME), 
+              getField(activity, DB.ACTIVITIES_END_TZ)), //timeDefault, 
+          this.timezoneDropdown('end')                   // tzpicker 
           )}
-        {formElements.textElementFormGroup(
+        {formElements.textElementFormGroup( // DESCRIPTION
             'formActivityDescription', // controlId
             'Description:',            // formLabel
```

Comment:
Align comments.

Reply:
it's more useful if theyre always closer